specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-02-01-preview/desktopvirtualization.json

```diff
@@ -4281,6 +4281,11 @@
       "type": "object",
       "description": "Scaling plan properties.",
       "properties": {
+        "ring": {
+          "type": "integer",
```

Comment:
Might it better be an enum? (depends on your domain knowledge here, i.e., will it be ring 16 there?)

Reply:
I think integer makes the most sense. We are considering adding more rings in the near future, and keeping with int makes the scalingPlan resource match the existing hostPool resource, which uses an integer for its ring value.